### PR TITLE
Dx 618

### PR DESCRIPTION
**Pull request by jmulford-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `gitbook` 

- [ ] I made sure that the site looks great and functions perfectly live
- [ ] I ran splell check
- [ ] I'm ready for these changes to be made live

### General Review

- [ ] All changes/updates requested were intentionally changed or added (no extraneous file or partial updates)
- [ ] The live rendered page behaves and looks like intended.
- [ ] All links work as intended. (click every link to make sure that it takes the user to the expected place).
- [ ] The markdown was rendered to HTML as intended. (Tables look right, code samples are gated properly).
- [ ] Any style or CSS changes are tested on multiple different pages to ensure nothing unexpected broke or changed.
- [ ] I proof-read the live site (`gitbook serve`) and there are no oblivious splelling or grammer misteaks.

### Gitbook Specific

- [ ] Any new documentation pages have been added to the `SUMMARY.md` file so they are rendered to HTML.
- [ ] Any links to other pages within this documentation use relative links (`[read more about ...](guides/voice/voicemail.md)`).
- [ ] Gitbook can install and build the documentation (`gitbook install` & `gitbook build`)

### Code Specific

- [ ] All code has been tested against the live API.
- [ ] All code can be run without throwing errors or **new** warnings.
- [ ] All code is formatted correctly and consistently (spaces, tabs).
- [ ] All code is legible and idiomatic to the language.
- [ ] All code uses sensical method, function, and variable names.
- [ ] All code samples use the appropriate syntax highlighting.
- [ ] Any examples re-use same phone numbers, uuids, variable names, etc... throughout (IE don't have something like `{from: '+19191231234', to: '+19191231234'}` unless it's meant to show the same phone number calling/texting itself ).

## TL;DR

- [ ] I did these things
- [ ] I'm ready for these changes to be made live

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-618.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-618.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-618.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-618.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-618.s3-website-us-east-1.amazonaws.com/

File: README.md [hunk 1]
```diff
@@ -71,7 +71,22 @@ curl --request POST \
...
 
 ```python
-## Coming Soon
+from bandwidth.bandwidth_client import BandwidthClient
+from bandwidth.messaging.models.message_request import MessageRequest
+
+bandwidth_client = BandwidthClient(
+    messaging_basic_auth_user_name=messaging_basic_auth_user_name,
+    messaging_basic_auth_password=messaging_basic_auth_password)
+messaging_client = bandwidth_client.messaging_client.client
```
Comment (ghost) [inline README.md]: For brevity sake, we can leave out initialization from the Readme snips.

File: README.md [hunk 2]
```diff
@@ -104,7 +119,22 @@ curl --request POST \
...
 
 ```python
-## Coming Soon
+from bandwidth.bandwidth_client import BandwidthClient
+from bandwidth.voice.models.api_create_call_request import ApiCreateCallRequest
+
+bandwidth_client = BandwidthClient(
+    voice_basic_auth_user_name=voice_basic_auth_user_name,
+    voice_basic_auth_password=voice_basic_auth_password)
+voice_client = bandwidth_client.voice_client.client
```
Comment (ghost) [inline README.md]: For brevity sake, we can leave out initialization from the Readme snips

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-618.s3-website-us-east-1.amazonaws.com/

**Review by ghost — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-618.s3-website-us-east-1.amazonaws.com/

**Review by ghost — APPROVED** [review]

**Pull request merged**